packages/fx-core/src/plugins/resource/frontend/dotnet/constants.ts

```diff
@@ -84,3 +96,11 @@ export class WebappBicep {
     domainAsParam: WebappBicep.domainAsParam,
   };
 }
+
+export class AppSettingsPlaceholder {
+  static readonly clientId = "$clientId$";
+  static readonly clientSecret = "$client-secret$";
```

Comment:
Nit: Naming conventions are inconsistent.
`clientId`, `botId` vs `client-secret`, `bot-password`

Reply:
Also curious why the naming conventions are different?